js/flightlog_fields_presenter.js

```diff
@@ -1128,6 +1162,47 @@ function FlightLogFieldPresenter() {
                         default:
                             return value.toFixed(0) + 'us';
                     }
+                case 'GPS_RESCUE_THROTTLE_PID':
+                    switch (fieldName) {
+                        case 'debug[0]': // Throttle P added uS
+                        case 'debug[1]': // Throttle D added * uS
+                            return value.toFixed(0) + 'uS';
+                        case 'debug[2]': // current altitude in m
+                        case 'debug[3]': // TARGET altitude in m
+                            return (value / 100).toFixed(1) + 'm';
+                        default:
+                            return value.toFixed(0);
+                    }
+                case 'GPS_RESCUE_VELOCITY':
+                    switch (fieldName) {
+                        case 'debug[0]': // Pitch P degrees * 100
+                        case 'debug[1]': // Pitch D degrees * 100
+                            return (value / 100).toFixed(1) + 'deg';
+                        case 'debug[2]': // velocity to home cm/s
+                        case 'debug[3]': // velocity target cm/s
+                            return (value / 100).toFixed(1) + 'm/s';
```

Comment:
```suggestion
                            return (value / 100).toFixed(1) + 'm/s';
                        default:
                            return value.toFixed(0);
```

We need an unconditional check in java script.